addons/auth_passkey/models/auth_passkey_key.py

```diff
@@ -60,7 +60,7 @@ def _inverse_public_key(self):
     def _get_session_challenge(self):
         challenge = request.session.pop('webauthn_challenge', None)
         if not challenge:
-            raise AccessDenied('Cannot find a challenge for this session')
+            raise AccessDenied(_('Cannot find a challenge for this session'))
```

Comment:
Should `AccessDenied` errors be translated? Most of them in this PR seem rather technical, translating it might make it harder to troubleshoot issues.